### PR TITLE
Refactor the shell code

### LINKED ISSUE #1: Rearrange and simply shell structure
Reorganize the shell/ directory like protoshell.

### PR DESCRIPTION
**Pull request by iBeliever**
Fixes #1

### CODE REVIEW

**Comment by iBeliever** [comment]
@plfiorini This is ready for review now.

File: shell/base/DesktopOutputWindow.qml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.0
```
Comment (plfiorini) [inline shell/base/DesktopOutputWindow.qml]: missing license

Comment (iBeliever) [inline shell/base/DesktopOutputWindow.qml]: Fixed.

**Comment by plfiorini** [comment]
I'll test it today thanks

**Pull request merged**